### PR TITLE
replaces mailhog with mailpit

### PR DESCRIPTION
**Pull request by shelane**
Removing the abandoned mailhog with mailpit and updating the yaml file based on our recommended extends approach.

### CODE REVIEW

**Review by lmakarov — CHANGES_REQUESTED** [review]
`PHP_INI` and `MAILHOG_INI` are no longer relevant for `mailpit` (nor where they for `mailhog`) since this change in `docksal/cli` - https://github.com/docksal/service-cli/issues/103

@shelane Can you please update the mailpit addon to remove the unnecessary code?

**Comment by shelane** [comment]
Yeah, I had removed that. The mailpit directory does not contain the ini file and the whole mailhog directory is being deleted with this request.

**Comment by shelane** [comment]
Ah, I see where you mean. Fixing now.

File: mailpit/mailpit.post-install [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+fin mailpit enable
+
+echo -e "${green}Mailpit UI:${NC} http://mail.<project_name>.docksal"
+echo -e "${green}More documentation:${NC} http://docs.docksal.io/en/master/tools/mailpit/"
```
Comment (lmakarov) [inline mailpit/mailpit.post-install]:
```suggestion
echo -e "${green}More documentation:${NC} https://docs.docksal.io/service/other/mailpit/"
```

File: mailpit/mailpit [hunk 2]
```diff
@@ -171,13 +139,13 @@ cd "$PROJECT_ROOT"
 
 case "$1" in
 	enable)
-		mailhog_enable
+		MAILPIT_enable
```
Comment (lmakarov) [inline mailpit/mailpit]:
```suggestion
		mailpit_enable
```

File: mailpit/mailpit [hunk 3]
```diff
@@ -171,13 +139,13 @@ cd "$PROJECT_ROOT"
...
 		;;
 	disable)
-		mailhog_disable
+		MAILPIT_disable
```
Comment (lmakarov) [inline mailpit/mailpit]:
```suggestion
		mailpit_disable
```

File: mailpit/mailpit [hunk 4]
```diff
@@ -92,74 +90,44 @@ yml_remove_service ()
...
-			# Remove all lines that present in mailhog.ini from php.ini
-			php_ini_contents=${php_ini_contents/$mailhog_ini_contents/}
-			echo -e "$php_ini_contents" > "$PHP_INI"
-			;;
-	esac
-}
 
 # Enable container and settings
-mailhog_enable ()
+MAILPIT_enable ()
```
Comment (lmakarov) [inline mailpit/mailpit]:
```suggestion
mailpit_enable ()
```

File: mailpit/mailpit [hunk 5]
```diff
@@ -92,74 +90,44 @@ yml_remove_service ()
...
+	# Add mailpit service to docksal.yml
+	yml_add_service "$MAILPIT_YML"
 	# Apply stack changes.
 	fin stop cli
 	fin up
 }
 
 # Disable container and settings
-mailhog_disable ()
+MAILPIT_disable ()
```
Comment (lmakarov) [inline mailpit/mailpit]:
```suggestion
mailpit_disable ()
```

**Review by lmakarov — CHANGES_REQUESTED** [review]

**Review by lmakarov — APPROVED** [review]

**Pull request merged**